### PR TITLE
boxType system is ever $isMultilingual = true

### PR DESCRIPTION
**Pull request by Griborim**
Boxtype system is always multi-lingual about the acp, so it should also be about the pip so.

### CODE REVIEW

File: wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php [hunk 1]
```diff
@@ -179,12 +179,13 @@ protected function prepareImport(array $data) {
...
 				}
 				
-				if (count($data['elements']['content']) === 1) {
+				if (count($data['elements']['content']) === 1 && $boxType != 'system') {
 					if (!isset($data['elements']['content'][''])) {
 						throw new SystemException("Expected one 'content' element without a 'language' attribute for box '{$identifier}'");
 					}
-				}
-				else {
+				} else if (isset($data['elements']['content']['']) && $boxType == 'system') {
```
Comment (joshuaruesweg) [inline wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php]: code style - - must be: 
```
}
else if (/*…*/) {
```

File: wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php [hunk 2]
```diff
@@ -179,12 +179,13 @@ protected function prepareImport(array $data) {
...
+					throw new SystemException("Expected one 'content' element with a 'language' attribute for box '{$identifier}'");
+				} else {
```
Comment (joshuaruesweg) [inline wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php]: code style - must be: 
```
}
else {
```

**Comment by dtdesign** [comment]
This code makes absolutely no sense.

File: wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php (hunk 2 above)
Comment (Griborim) [inline wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php]: Thanks, the code style is gefixt.

File: wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php (hunk 1 above)
Comment (Griborim) [inline wcfsetup/install/files/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php]: Thanks, the code style is gefixt.

**Comment by TimWolla** [comment]
> The changes above are within a branch of the switch that explicitly targets a boxType != system

@dtdesign 
You seem to have missed the fall through. No idea whether this changes anything.

**Comment by Griborim** [comment]
I have test with this code.

`        <box identifier="de.incendium.linklist.Categories">
            <name language="de">Kategorien (Links)</name>
            <boxType>system</boxType>
            <objectType>de.incendium.linklist.categories</objectType>
            <position>sidebarRight</position>
            <showOrder>1</showOrder>
            <showHeader>1</showHeader>
            <visibleEverywhere>0</visibleEverywhere>
            <visibilityExceptions>
                <page>de.incendium.linklist.EntryList</page>
            </visibilityExceptions>
            
            <content language="de">
                <title>Kategorien</title>
            </content>
        </box>`

And i became this error.

I Think you have forgot a break; in the line 170?


Error Type:

wcf\system\exception\SystemException

Error Message:

Expected one 'content' element without a 'language' attribute for box 'de.incendium.linklist.Categories'

File:

/var/www/dev/lib/system/package/plugin/BoxPackageInstallationPlugin.class.php (184)

**Comment by dtdesign** [comment]
> And i became this error.
You just mixed up `became` (= `wurde`) and `got` (= `bekommen`) ;)

In other news, the commit above should fix the issue by allowing non-i18n content for system boxes while still flagging them as i18n boxes.

**Pull request closed without merging**